lib/validic/client.rb

```diff
@@ -44,8 +44,8 @@ class Client
     #
     # @params options[Hash]
     def initialize(options={})
-      @api_url          = options.fetch(:api_url, 'https://api.validic.com')
-      @api_version      = options.fetch(:api_version, 'v1')
+      @api_url          = options.fetch(:api_url, Validic.api_url)
```

Comment:
I don't see api_url being set to a default anywhere, so won't this now default to nil instead of the url above? Seems pretty non passive unless I missing where its set. 
The tests appear to set this by default which might be leading to some false positive tests
https://github.com/Validic/validic/blob/40773d15bb62f33de829b58062dbdba61d691057/spec/spec_helper.rb#L9